integreat_cms/cms/forms/events/event_translation_form.py

```diff
@@ -96,13 +96,13 @@ def clean_slug(self):
         self.data["slug"] = unique_slug
         return unique_slug
 
-    def clean_description(self):
+    def clean_content(self):
         """
-        Validate the description field (see :ref:`overriding-modelform-clean-method`) and applies changes to <img>- and <a>-Tags to match the guidelines.
+        Validate the content field (see :ref:`overriding-modelform-clean-method`) and applies changes to <img>- and <a>-Tags to match the guidelines.
 
-        :raises ~django.core.exceptions.ValidationError: When a heading 1 (``<h1>``) is used in the description
+        :raises ~django.core.exceptions.ValidationError: When a heading 1 (``<h1>``) is used in the content
 
-        :return: The valid description
+        :return: The valid content
         :rtype: str
         """
-        return self.content_clean_method("description")
+        return self.content_clean_method("content")
```

Comment:
Since all content forms now have the same `content` field, you can delete this method and rename [CustomContentModelForm.content_clean_method()](https://github.com/digitalfabrik/integreat-cms/blob/d540cc3af8b94035f79f696f74f2800d4495686e/integreat_cms/cms/forms/custom_content_model_form.py#L25) to `clean_content`.